SourceX/storm/storm.cpp

```diff
@@ -485,6 +488,11 @@ BOOL SVidPlayBegin(char *filename, int a2, int a3, int a4, int a5, int flags, HA
 		deviceId = SDL_OpenAudioDevice(NULL, 0, &audioFormat, NULL, 0);
 		if (deviceId == 0) {
 			SDL_Log(SDL_GetError());
+#ifdef SWITCH
+			Mix_OpenAudio(22050, AUDIO_S16LSB, 2, 1024);

```

Comment:
Probably redundant if `SDL_Init` is reverted to also enable audio.

Reply:
On Switch at least, this is still necessary even after fixing SDL_Init. SDL_Mixer docs specifically forbid calling SDL_OpenAudioDevice while using mixer.